frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -126,20 +128,42 @@ export const ReportStreamHeader = () => {
             );
         }
 
-        /* ADMIN ONLY */
+        /* ADMIN ONLY 
+        
+          Build a drop down for file handler links
+        */
         if (memberships.state.active?.memberType === MemberType.PRIME_ADMIN) {
             // Validate NavLink
-            if (CheckFeatureFlag(FeatureFlagName.VALIDATION_SERVICE)) {
+            const features = [
+                {
+                    access: CheckFeatureFlag(
+                        FeatureFlagName.VALIDATION_SERVICE
+                    ),
+                    slug: "validate",
+                    title: "Validate",
+                },
+                {
+                    access: CheckFeatureFlag(FeatureFlagName.USER_UPLOAD),
+                    slug: "user-upload",
+                    title: "User Upload",
+                },
+            ];
+            if (features.some(({ access }) => access)) {
+                const fileHandlerDirectories = features.reduce(
+                    (acc, { access, title, slug }) => {
+                        if (access) {
+                            acc.push({ title, slug });
+                        }
+                        return acc;
+                    },
+                    [] as NonStaticOption[]
```

Comment:
Interested if you think this is something worth `TODO - integrate with DropdownNav tools` and throw it in a tech-debt item? 🤔 Feature flags built into the dropdown generator would be a good move.

Reply:
yeah, since feature flags seem to largely be used to activate / decativate routes and nav items, I think it's worth making that all more directly integrated. Here's a ticket https://app.zenhub.com/workspaces/experience-607d9d5e68b95200150fec37/issues/cdcgov/prime-reportstream/6177